<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SfcNew</class>
 <widget class="QDialog" name="SfcNew">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Input</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
background-color:#c5c5c5;
}</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget
{
background-color:#3f3f3f;
font: 10pt &quot;Times New Roman&quot;;
color:#fff;
}

QPushButton
{
background-color: #a19e9d;
}

QComboBox
{
background-color: #2a3914;
}</string>
   </property>
   <widget class="QComboBox" name="source">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>File</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Random</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Source:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="fromfile">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>281</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Filename</string>
     </property>
    </widget>
    <widget class="QPushButton" name="br_file">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="file">
     <property name="geometry">
      <rect>
       <x>62</x>
       <y>30</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="accept_but">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>230</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel_but">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>230</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="repeat">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Repeat</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="fromrandom">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>281</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Random</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Points</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="random">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
    </widget>
   </widget>
   <zorder>fromrandom</zorder>
   <zorder>fromfile</zorder>
   <zorder>source</zorder>
   <zorder>label</zorder>
   <zorder>accept_but</zorder>
   <zorder>cancel_but</zorder>
   <zorder>repeat</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
